### PR TITLE
New: Added initWithContentsOfURL: to CPImage.j

### PR DESCRIPTION
**Pull request by robert-plasq**
New : Added initWithContentsOfURL: 
It seemed that there was no simple way to create a CPImage from a URL, but that is not the case.

### CODE REVIEW

File: AppKit/CPImage.j [hunk 1]
```diff
@@ -208,6 +208,17 @@ function CPAppKitImage(aFilename, aSize)
...
 }
 
+/*!
+    Initializes and returns an image object with the contents of the specified URL.
+    @param url the URL representing the image
+    @return the initialized image
+*/
+- (id)initWithContentsOfURL:(CPURL)url
+{
+    return [self initWithContentsOfFile: [url absoluteString]];
```
Comment (daboe01) [inline AppKit/CPImage.j:218]: no whitespace after colon

**Comment by daboe01** [comment]
-#new
+AppKit

**Comment by daboe01** [comment]
+#needs-review

File: AppKit/CPImage.j (hunk 1 above)
Comment (robert-plasq) [reply AppKit/CPImage.j]: Removed the space and committed the fix to the branch.

**Comment by daboe01** [comment]
@robert-plasq please file one PR per issue. see https://github.com/cappuccino/cappuccino/blob/master/CONTRIBUTING.md#commit-messages

**Comment by robert-plasq** [comment]
Yeah - I didn't realize that further commits would be added to the outstanding pull request. So I'll abandon this one, then. I doubt it was ever going to get merged anyway.

**Pull request closed without merging**